### ISSUE TITLE
Test failing with aiohttp 3.10.1

### ISSUE THREAD

**Issue by mweinelt** [reefab/foobot_async#6]
Hi!

We're seeing the following test failure after upgrading aiohttp to the 3.10 release series with foobot-async 1.0.0:

```
___________ ERROR collecting foobot_async/tests/test_foobot_async.py ___________
foobot_async/tests/test_foobot_async.py:9: in <module>
    client = FoobotClient('token', 'example@example.com')
foobot_async/__init__.py:42: in __init__
    self._session = aiohttp.ClientSession()
/nix/store/ahq3bc6pmnzmwgic0gap5ma75jajr7j8-python3.12-aiohttp-3.10.1/lib/python3.12/site-packages/aiohttp/client.py:296: in __init__
    loop = loop or asyncio.get_running_loop()
E   RuntimeError: no running event loop
```

**Comment by reefab** [comment]
Got it, I'll look into it.

**Comment by reefab** [comment]
Tests fixed in 1.0.1.

**Comment by mweinelt** [comment]
Thanks!
